single_stage_detector/requirements.txt

```diff
@@ -1,14 +1,5 @@
-Cython==0.28.4
-git+git://github.com/NVIDIA/apex.git@9041a868a1a253172d94b113a963375b9badd030#egg=apex
-mlperf-compliance==0.0.10
-cycler==0.10.0
-kiwisolver==1.0.1
-matplotlib==2.2.2
-numpy==1.19.1
-Pillow==5.2.0
-pyparsing==2.2.0
-python-dateutil==2.7.3
-pytz==2018.5
-six==1.11.0
-torchvision==0.2.1
-pycocotools==2.0.2
+scikit-image>=0.15.0
+ujson>=4.0.2
+matplotlib>=3.5.1
+pycocotools==2.0.0
+git+https://github.com/mlcommons/logging.git@1.1.0-rc4
```

Comment:
Just a note that the logger version will need to be updated once v2.0 is finalized

RCP and compliance PR in the logging repo: https://github.com/mlcommons/logging/pull/209